DICOMPlugins/DICOMM3DPlugin.py

```diff
@@ -0,0 +1,174 @@
+from __future__ import absolute_import
+import glob
+import os
+import json
+import vtk
+import io
+import shutil
+import vtkSegmentationCorePython as vtkSegmentationCore
+import logging
+
+from base.DICOMM3DPluginBase import DICOMM3DPluginBase
+
+import slicer
+from DICOMLib import DICOMLoadable
+
+from SlicerDevelopmentToolboxUtils.mixins import ModuleLogicMixin
+from SlicerDevelopmentToolboxUtils.constants import DICOMTAGS
+from six.moves import range
+
+
+#
+# This is the plugin to handle translation of DICOM SEG objects
+#
+class DICOMM3DPluginClass(DICOMM3DPluginBase):
+
+  def __init__(self):
+    super(DICOMM3DPluginClass,self).__init__()
+    self.loadType = "DICOMM3D"
+
+  def examineFiles(self,files):
+    """ Returns a list of DICOMLoadable instances
+    corresponding to ways of interpreting the
+    files parameter.
+    """
+    loadables = []
+
+    # just read the modality type; need to go to reporting logic, since DCMTK
+    #   is not wrapped ...
+
+    for cFile in files:
+
+      uid = slicer.dicomDatabase.fileValue(cFile, self.tags['instanceUID'])
+      if uid == '':
+        return []
+
+      desc = slicer.dicomDatabase.fileValue(cFile, self.tags['seriesDescription'])
+      if desc == '':
+        desc = "Unknown"
+      
+      # EncapsulatedDocument = slicer.dicomDatabase.fileValue(cFile, self.tags['EncapsulatedDocument'])
+      # EncapsulatedDocumentLength = slicer.dicomDatabase.fileValue(cFile, self.tags['EncapsulatedDocumentLength'])
```

Comment:
More style suggestions: we avoid leaving in 'dead code' unless it's something that future developers might need for debugging, and if so an explicit comment describing that use is good practice.